crates/rolldown/tests/fixtures/cjs_compat/reexport_commonjs/artifacts.snap

```diff
@@ -39,6 +39,6 @@ assert.equal(foo_ns.foo, undefined);
 assert.equal((Object.keys(foo_ns)).length, 2);
```

Comment:
This is unexpected, I would have a look, but I think it doesn't block the merging. since this is a hotfix

Reply:
tracked https://github.com/rolldown/rolldown/issues/1715